<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Thread</class>
 <widget class="QWidget" name="Thread">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thread</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>239</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonStop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonQuit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
